database/postgres/dal/user_subscription.py

```diff
@@ -0,0 +1,39 @@
+from database.models import User, TempSubscribersData, UserClubPreferences
+
+class UserSubscriptionDAL:
+    def __init__(self, db_session):
+        self.db_session = db_session
+
+    def try_add_subscribers_data(self, preference_index):
+        users = self.__get_users_by_preference_index(preference_index)
+
+        new_rows = [
+            TempSubscribersData(team_ids=preference_index, subscriber_emails=user.email)
```

Comment:
`team_ids = preference_index`. Looks a little bit odd. Could you please elaborate?